src/Authentication/Authentication/test/Microsoft.Graph.Authentication.Tests.ps1

```diff
@@ -51,7 +51,8 @@ Describe "Microsoft.Graph.Authentication module" {
                     "Invoke-MgRestMethod",
                     "Get-MgRequestContext",
                     "Set-MgRequestContext",
-                    "Set-MgGraphOption"
+                    "Set-MgGraphOption",
+                    "Get-MgGraphOption"
```

Comment:
Let's add Pester tests for `Set-MgGraphOption` and `Get-MgGraphOption`.  See example at https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/features/2.0/src/Authentication/Authentication/test/Disconnect-MgGraph.Tests.ps1.